packages/smooth_app/lib/pages/product/product_page.dart

```diff
@@ -375,6 +375,16 @@ class _ProductPageState extends State<ProductPage> {
           i < _product.categoriesTags!.length;
           i++) {
         final String categoryTag = _product.categoriesTags![i];
+        final String currentLanguageCode =
+            ProductQuery.getCurrentLanguageCode(context);
+        final OpenFoodFactsLanguage currentLanguage =
+            LanguageHelper.fromJson(currentLanguageCode);
+        String categoryTagInLocalLanguage = categoryTag;
```

Comment:
sugg:
```dart
        final String categoryTagInLocalLanguage =
            _product.categoriesTagsInLanguages?[currentLanguage]?[i] ?? categoryTag;
```

Reply:
Thanks, that looks much better. I had forgotten about the ? operator.